client/scss/components/_cookie_notification.scss

```diff
@@ -0,0 +1,68 @@
+//* @define cookie-notification
+
+.cookie-notification {
+  @include font('HNM6');
+
+  @include respond-to('medium') {
+    @include font('HNM4');
+  }
+
+  display: none;
+  background: color('teal');
+  color: color('white');
+  padding: $vertical-space-unit 0;
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  z-index: 2;
+  opacity: 1;
+  transition: opacity 600ms ease;
+
+  .icon {
+    width: 1.3em;
+    min-width: 1.3em;
+  }
+
+  .icon__shape {
+    fill: color('white');
+  }
+
+  &.cookie-notification--is-visible {
+    display: block;
+  }
+
+  &.cookie-notification--is-faded {
+    opacity: 0;
```

Comment:
Good point. I'll change the z-index after a transition-delay so that the fade is visible.